src/OrbitGl/CMakeLists.txt

```diff
@@ -155,6 +157,7 @@ target_link_libraries(
          OrbitAccessibility
          OrbitPaths
          PresetFile
+         Qt5::Core
```

Comment:
You're importing `Qt5::Core`, but you're using `QApplication` which is part of `Qt5::Widgets`. So I guess you need to link to the latter instead. `setOverrideCursor` is technically part of `QGuiApplication` (parent of `QApplication`) and `QGuiApplication` lives in Qt5::Gui. So if you want to minimize the linking surface as much as possible you could also link against `Qt5::Gui` and use `QGuiApplication` instead of `QApplication`  in the code below.

Reply:
Good point, thanks. I changes this as suggested.